### ISSUE TITLE
Support for RAISE and COMMENT

### ISSUE THREAD

**Issue by gabssnake** [oguimbal/pg-mem#51]
These might not be used by too many people, but we won't mind adding them if you're OK with that?

* [`RAISE`](https://www.postgresql.org/docs/8.3/plpgsql-errors-and-messages.html)
```sql
RAISE NOTICE 'Some message here';
```

* [`COMMENT`](https://www.postgresql.org/docs/current/sql-comment.html)
```sql
CREATE TABLE groups (members VARCHAR);
COMMENT ON COLUMN groups.members is 'Nice column you have there';
```

**Comment by gabssnake** [comment]
You're awesome ! Thank you very much !

**Comment by oguimbal** [comment]
This should be fixed with `pg-mem@1.6.1` 

NB: Comments & notices are juste ignored altogeter by the execution engine.
